monitoring/ceph-mixin/prometheus_alerts.yml

```diff
@@ -653,7 +653,7 @@ groups:
           ) >= 0.0001 or (
             increase(node_network_receive_drop_total{device!="lo"}[1m]) +
             increase(node_network_transmit_drop_total{device!="lo"}[1m])
-          ) >= 10
+          ) >= 600
```

Comment:
You need to change the value in the alert description as well - 

`Node {{ $labels.instance }} experiences packet drop > 0.01% or >
            600 packets/s on interface {{ $labels.device }}.`